x-pack/plugins/actions/server/lib/action_executor.ts

```diff
@@ -486,14 +526,17 @@ function validateAction(
         secrets,
       });
     }
+    if (requeueInvalidTasksConfig?.enabled) {
```

Comment:
It feels confusing to me that we're passing the entire `requeueInvalidTasksConfig` into the task runner for each task type. Task types shouldn't need to know what the configured max attempts or the delay is. Could we just pass a boolean flag like `shouldValidateParams`?

Reply:
done